install.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 
 if [[ -z $OPOSSUM_HEADLESS_SETUP ]]; then
+    BOOST_INSTALLED=$(dpkg-query -W --showformat='${Status}\n' libboost1.67-dev 2>/dev/null | grep "install ok installed")
```

Comment:
Maybe add a comment that this is now the place in our install script where we may add necessary upfront cleanup steps.

Reply:
Hopefully this won't become a common pattern

Reply:
> Hopefully this won't become a common pattern

Agreed. Anyhow it saves people time to know how to deal with such problems.